message_converter/src/converter_node.cpp

```diff
@@ -244,7 +229,7 @@ int main(int argc, char **argv)
     // Values in comments are from the parking lot test
     // double home_latitude = 42.3711071377; 
     // double home_longitude = -71.2168400352; 
-    // double home_altitude = -15.3775837652; 
+    // double home_altitude = -15.3775837652;     
     SwiftNavRover obj(&n, home_latitude, home_longitude, home_altitude);
     ros::spin();
```

Comment:
`ros::spin()` is blocking, so it might make sense to replace this with a  loop that calls `spinOnce()` and sleeps

Reply:
Fixed in https://github.com/Humatics/ethz_piksi_ros/pull/1/commits/76e26918e9f52116b26f10a1cd0a0ce386e9d8db